api/turing/api/routers_api.go

```diff
@@ -35,6 +35,20 @@ func (c RoutersController) ListRouters(
 		return InternalServerError("unable to list routers", err.Error())
 	}
 
+	// Append monitoring URL to router
+	for _, router := range routers {
+		monitoringURL, err := c.RouterVersionsService.GenerateMonitoringURL(
```

Comment:
Same comment about whether service is a better place to implement this than controller

Reply:
done